projects/plugins/jetpack/sal/class.json-api-site-base.php

```diff
@@ -1733,4 +1733,13 @@ public function garden_name() {
 	public function garden_partner() {
 		return null;
 	}
+
+	/**
+	 * Detect whether the Garden site is provisioned.
+	 *
+	 * @return bool
+	 */
+	public function garden_is_provisioned() {
+		return false;
```

Comment:
I'm not sure if this could be an issue, but the default value you set on the Jetpack SAL, is null. Shouldn't we return null here too?

192999-ghe-Automattic/wpcom/files#diff-8d8698feb541138c2f94860172cf76a6a99b9ca24a2c6f8dd5c8aba2713fccfaR354-R360